msteams-platform/m365-apps/prerequisites.md

```diff
@@ -25,15 +25,26 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
 
 You'll also need to enable sideloading for your tenant:
 
-1. Sign in to Microsoft 365 admin center (https://admin.microsoft.com) with your test tenant credentials and select **Teams** from the side panel to open the *Microsoft Teams admin center*.
-1. Select: Teams apps > Manage apps > **Org-wide app settings**.
-1. Under **Custom apps**, toggle on the option *Interaction with custom apps*.
+ 1. Sign in to [Microsoft 365 admin center](https://admin.microsoft.com/Adminportal/Home#/homepage) with your test tenant credentials.
 
-    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="Enable sideloading for custom apps from the Teams admin center":::
+ 1. From the left pane, select **Show all**.
+
+ 1. Under admin centers, select **Teams**. A new tab opens the Microsoft Teams admin center.
+
+ 1. Go to **Teams apps** > **Manage apps**.
+
+ 1. At the top right, select **Org-wide app settings**.
+
+ 1. Under Custom apps, turn on the **Interaction with custom app** toggle.
+
+    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="The screenshot is an example that enables sideloading for custom apps from the Teams Admin Center":::
+
+> [!NOTE]
+> Apart from Org-wide app settings, App setup policies also allow users to upload custom apps to Teams. If you are unable to upload custom apps after updating the Org-wide app settings, verify if the **Upload custom apps** option is enabled in **App setup policies**. For more information, see [Manage app setup policies](/microsoftteams/teams-app-setup-policies).
```

Comment:
Why is this a Note? We should document the full process here. If app setup policy is required we should say so.

Reply:
Removed the note as suggested.